configs/sites/frontera/compilers.yaml

```diff
@@ -0,0 +1,56 @@
+compilers::
+  - compiler:
+      spec: intel@=23.1.0
+      paths:
+        cc: /opt/intel/oneapi/compiler/2023.1.0/linux/bin/intel64/icc
+        cxx: /opt/intel/oneapi/compiler/2023.1.0/linux/bin/intel64/icpc
+        f77: /opt/intel/oneapi/compiler/2023.1.0/linux/bin/intel64/ifort
+        fc: /opt/intel/oneapi/compiler/2023.1.0/linux/bin/intel64/ifort
+      flags:
+        cflags: -diag-disable=10441
+        cxxflags: -diag-disable=10441
+        fflags: -diag-disable=10448
+      operating_system: centos7
+      target: x86_64
+      modules:
+      - intel/23.1.0
+      environment:
+        prepend_path:
+          PATH: '/opt/apps/gcc/9.1.0/bin'
+          CPATH: '/opt/apps/gcc/9.1.0/include'
+          LD_LIBRARY_PATH: '/opt/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin:/opt/apps/gcc/9.1.0/lib64:/opt/apps/gcc/9.1.0/lib'
+        set:
+          # https://github.com/JCSDA/spack-stack/issues/1012
+          I_MPI_EXTRA_FILESYSTEM: 'ON'
+          # https://github.com/JCSDA/spack-stack/issues/1011
```

Comment:
Do you know if you need lines 25 to 41? These tuning parameters are specific to Discover's new AMD Milan system with their bespoke parallel filesystem.

Reply:
@climbfuji Okay. I removed them and the spack-stack installation is fine with it. I still did not test UFS Weather Model or UFS Coastal with this installation but I'll try.